px-data-grid.html

```diff
@@ -587,17 +587,20 @@
             },
 
             /**
-             * Default column width if not defined, eg. '100px'
+             * Default column width if not defined, eg. '100px'. Notice that with not
+             * zero flex values this value behaves more as minimum width. See
+             * defaultColumnFlexGrow property.
              */
             defaultColumnWidth: {
               type: String,
               value: '100px'
             },
 
             /**
-             * Default column flex if not defined, eg. 1
+             * Default column flex if not defined, eg. 1. Equialent to the CSS flex-grow
+             * property.
              */
-            defaultColumnFlex: {
+            defaultColumnFlexGrow: {
               type: Number,
               value: 1
```

Comment:
It's here @limonte

Reply:
Ups, I jumped files. px-data-grid-paginated is using two way binding, so it can just use the default from px-data-grid.